### DISCUSSION TITLE
UI Issue Polkadot Explorer

### DISCUSSION THREAD

**Discussion by kw1knode** [subquery/subql#925] [Frontier TestNet - Bug Reporting]
Wrong deployment when visiting Polkadot Subquery Project

https://testnet.thechaindata.com/explorer/project/0x00/indexers

When loaded it displays Deployment ID `QmYR8xQgAXuCXMPGPVxxR91L4VtKZsozCM7Qsa5oAbyaQ3`  
![1](https://user-images.githubusercontent.com/25113494/165436209-b74a3455-e89e-46f8-a2db-206c8368f059.png)

However, if you select ver 0.0.1 again the correct deployment is displayed.
![3](https://user-images.githubusercontent.com/25113494/165436278-57454ff0-6c10-4404-9213-779836ec270a.png)

**Comment by mzxyz** [comment]
@CaiYiLiang
